python/cuml/tests/test_train_test_split.py

```diff
@@ -23,10 +23,11 @@
 cudf = gpu_only_import("cudf")
 cp = gpu_only_import("cupy")
 np = cpu_only_import("numpy")
+pd = cpu_only_import("pandas")
 
 cuda = gpu_only_import_from("numba", "cuda")
 
-test_array_input_types = ["numba", "cupy"]
+test_array_input_types = ["numba", "cupy", "cudf", "pandas"]
```

Comment:
Code looks great! But the in the tests we're only testing for dataframes, should we add tests for series as well?

Reply:
Existing tests have hardcoded 2-d array inputs that are can't be naturally cast to Series. I parametrize them with 1-d array and a fixture to convert to df / series based on data shape.